scripts/jenkins/Jenkinsfile

```diff
@@ -544,68 +553,92 @@ kill `pidof dbus-daemon`
 
 /* Generate Stages that build and deploy artifacts
  */
-def generate_artifact_stages() {
+def generateArtifactStages() {
     def tasks = [:]
-    tasks << build_package_debian_stretch()
-    tasks << build_homepage()
+    tasks << buildPackageDebianStretch()
+    tasks << buildHomepage()
     return tasks
 }
 
-def build_package_debian_stretch() {
-    def stage_name="package/stretch"
-    return [(stage_name): {
-        stage(stage_name) {
-            withDockerEnv(DOCKER_IMAGES.stretch) {
-                withCredentials([file(credentialsId: 'jenkins-key', variable: 'KEY'),
-                                 file(credentialsId: 'jenkins-secret-key', variable: 'SKEY')]) {
-                    sh "gpg --import $KEY"
-                    sh "gpg --import $SKEY"
-                }
-                withEnv(["DEBSIGN_PROGRAM=gpg",
-                         "DEBFULLNAME=Jenkins (User for Elektra automated build system)",
-                         "DEBEMAIL=autobuilder@libelektra.org"]) {
-                    sh "rm -R ./*"
-                    def target_dir="./libelektra"
-                    checkout scm: [
-                        $class: 'GitSCM',
-                        branches: scm.branches,
-                        extensions: scm.extensions + [
-                            [$class: 'PerBuildTag'],
-                            [$class: 'RelativeTargetDirectory',
-                             relativeTargetDir: target_dir]
-                        ],
-                        userRemoteConfigs: scm.userRemoteConfigs
-                    ]
-                    dir(target_dir) {
-                        sh "git checkout -B temp"
-                        sh "git tag -f $VERSION"
+def buildPackageDebianStretch() {
+    return withDockerEnv(DOCKER_IMAGES.stretch, [DOCKER_OPTS.MOUNT_MIRROR]) {
+        withCredentials([file(credentialsId: 'jenkins-key', variable: 'KEY'),
+                         file(credentialsId: 'jenkins-secret-key', variable: 'SKEY')]) {
+            sh "gpg --import $KEY"
+            sh "gpg --import $SKEY"
+        }
+        withEnv(["DEBSIGN_PROGRAM=gpg",
+                 "DEBFULLNAME=Jenkins (User for Elektra automated build system)",
+                 "DEBEMAIL=autobuilder@libelektra.org"]) {
+            sh "rm -R ./*"
+            def targetDir="./libelektra"
+            checkout scm: [
+                $class: 'GitSCM',
+                branches: scm.branches,
+                extensions: scm.extensions + [
+                    [$class: 'PerBuildTag'],
+                    [$class: 'RelativeTargetDirectory',
+                     relativeTargetDir: targetDir]
+                ],
+                userRemoteConfigs: scm.userRemoteConfigs
+            ]
+            dir(targetDir) {
+                sh "git checkout -B temp"
```

Comment:
Why not run (parts of) scripts/build-debian-package
(add commandline argument for which repo should be used and only download if not specified)

Reply:
again i only copied existing build jobs. if you want further modifications that should be done later.